plasmapy/classes/plasma.py

```diff
@@ -89,3 +96,93 @@ def alfven_speed(self):
         B = self.magnetic_field
         rho = self.density
         return np.sqrt(np.sum(B * B, axis=0) / (mu0*rho))
+
+
+class PlasmaBlob():
+    """
+    Class for describing and calculating plasma parameters without
+    spatial/temporal description.
+    """
+    def __init__(self, T_e, n_e, Z=None, particle='p'):
+        """
+        Initialize plasma paramters.
+        The most basic description is composition (ion), temperature,
+        density, and ionization.
+        """
+        self.T_e = T_e
+        self.n_e = n_e
+        self.particle = particle
+        if Z == None:
+            # extracting charge state from particle
+            self.Z = integer_charge(self.particle)
+        else:
+            self.Z = Z
+        # extract mass from particle
+        self.ionMass = ion_mass(self.particle)
+    def electron_temperature(self):
+        return self.T_e
+    def electron_density(self):
+        return self.n_e
+    def ionization(self):
+        return self.Z
+    def composition(self):
+        return self.particle
+    def regimes(self):
+        """
+        Generate a comprehensive description of the plasma regimes
+        based on plasma properties and consequent plasma parameters.
+        """
+        # getting dimensionless parameters
+        coupling = self.coupling()
+        quantum_theta = self.quantum_theta()
+
+        # determining regimes based off dimensionless parameters
+        # coupling
+        if coupling <= 0.01:
+            # weakly coupled
+            coupling_str = f"Weakly coupled regime: Gamma = {coupling}."
+        elif coupling >= 100:
+            # strongly coupled
+            coupling_str = f"Strongly coupled regime: Gamma = {coupling}."
+        else:
+            # intermediate regime
+            coupling_str = f"Intermediate coupling regime: Gamma = {coupling}."
+        # quantum_theta
+        if quantum_theta <= 0.01:
+            # Fermi energy dominant
+            quantum_theta_str = (f"Fermi quantum energy dominant: Theta = "
+                                 f"{quantum_theta}")
+        elif quantum_theta >= 100:
+            # thermal kinetic energy dominant
+            quantum_theta_str = (f"Thermal kinetic energy dominant: Theta = "
+                                 f"{quantum_theta}")
+        else:
+            # intermediate regime
+            quantum_theta_str = (f"Both Fermi and thermal energy important: "
+                                 f"Theta = {quantum_theta}")
+
+        # summarizing and printing/returning regimes
+        aggregateStrs = [coupling_str,
+                         quantum_theta_str,
+                         ]
+        print(aggregateStrs)
```

Comment:
Explicit print is probably bad here. Maybe it'd be best to return a multiline string `"\n".join(aggregateStrs)`? Maybe a dictionary of enums, in fact?

I'm probably rambling. This seems nice enough for now.

Reply:
I'm using this as a shortcut for printing. I've moved the printing to `__str__` overload. We can later implement a dictionary or somesuch under `regimes()` or another method.